main.js

```diff
@@ -0,0 +1,119 @@
+var weather = [];
+
+var fetchCoords = function(query) {
+
+  var url = `http://api.openweathermap.org/geo/1.0/direct?q=${query}&limit=1&appid=1620b57fab77c4f87fdf97f65eae66af`;
+  fetch(url, {
+    method: "GET",
+    dataType: "json"
+  })
+  .then(data => data.json())
+  .then(data => fetchData(data));
+}
+var fetchData = function(data) {
+
+  weather = [];
+  var lat = data[0].lat;
+  var lon = data[0].lon;
+  var urls = [`https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lon}&appid=1620b57fab77c4f87fdf97f65eae66af&units=imperial`, `https://api.openweathermap.org/data/2.5/forecast?lat=${lat}&lon=${lon}&appid=1620b57fab77c4f87fdf97f65eae66af&units=imperial`];
```

Comment:
appid is the same in both place so could create a variable for this to reduce the repeativeness